package.json

```diff
@@ -16,7 +16,7 @@
   "author": "André Abadesso <andre.abadesso@gmail.com>",
   "private": true,
   "devDependencies": {
-    "@types/jest": "^29.5.12",
+    "@types/jest": "29.5.13",
```

Comment:
This is funny, using `29.5.12` won't work. They probably removed it from the registry

We were actually using `29.5.13`